homeassistant/components/template/light.py

```diff
@@ -401,6 +468,21 @@ def _update_white_value(self, white_value):
             )
             self._white_value = None
 
+    @callback
+    def _update_effect_list(self, effect_list):
+        """Update the effect list from the template."""
+        try:
+            if effect_list in ("None", ""):
+                self._effect_list = None
+                return
+            self._effect_list = ast.literal_eval(effect_list)
```

Comment:
In 117 templates will return native typres. This is no longer necessary. 

You will also need to add a check that the return value is a list of strings. Anything else should be an error too.

Reply:
Ok, so i'll change those lines when i updgrate to 1.117

Reply:
Hi @balloob, i've updated code to use native types returned from templates